helm/charts/url-shortener/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
for now they should be since we are not setting up a helm repo, in any case the chart only includes the app definition similar to commiting cloudformation